### ISSUE TITLE
I want to embed Lightdash charts natively in Notion

### ISSUE THREAD

**Issue by hasan-rt** [lightdash/lightdash#3810]
### Description

How can I embed Lightdash analytics inside Notion?

### What problem does this solve?

I write technical specifications that include analytics.  I would like the numbers and graphs I include in my documents to stay current.

**Comment by owlas** [comment]
Hey @hasan-rt - great question!

I just tried embedding in Notion using a normal Lightdash link to a chart but it did not work well!

The problems we have (authenticating in embeds and allowing smaller screen sizes) are a general problem of iframe problems. I think we can leave this issue here and I can open a separate issue for embedding iframes into tools (such as notion or salesforce).

I'll also link the general embedding thread in case you wanted to make an API call: https://github.com/lightdash/lightdash/issues/1033

**Comment by owlas** [comment]
Yes @hasan-rt that's awesome and would make embedding much more native.

I've also opened an issue for generic embeds via iframes: https://github.com/lightdash/lightdash/issues/3897

**Comment by owlas** [comment]
Updating here with links. Anybody can build an Oauth integration with notion but being advertised in the directory is invite only:

- faqs: https://www.notion.so/integrations/all
- developer docs: https://developers.notion.com
- specific unfurl docs: https://developers.notion.com/page/link-previews-api

**Comment by jpetey75** [comment]
Updated the title since we do have embedding, but this issue is to track native embedding option within Notion

**Comment by torimwhaley** [comment]
relevant linear issue: https://linear.app/lightdash/issue/PROD-3097/i-want-to-embed-lightdash-charts-natively-in-notion

**Comment by jpetey75** [comment]
Embedding shipped after this was filed: you can generate a signed embed URL for a dashboard or chart ([#13836](https://github.com/lightdash/lightdash/pull/13836)) and paste it into a Notion embed block. There's no Notion-specific integration with live native embeds yet, so keeping this open for that. Let us know if the embed URL route works for your setup.
